pkg/ctl/delete/cluster.go

```diff
@@ -65,9 +73,14 @@ func doDeleteCluster(cmd *cmdutils.Cmd, force bool) error {
 		return err
 	}
 
+	// Ignore error here if cluster doesn't exist. We still would like to clean up the stack and other
+	// existing resources which were created even if the cluster failed.
 	cluster, err := cluster.New(cfg, ctl)
 	if err != nil {
-		return err
+		if !force {
```

Comment:
This might actually not be needed. I'm running a new test without it, since inside we already ignore if the refresh failed because the cluster didn't exist.

Reply:
Yep, this wasn't actually needed.